bounties/npm/node-sfml/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-17",
+	"AffectedVersionRange": "*",
+	"Summary": "Resources Downloaded over Insecure Protocol",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "sfml",
+		"URL": "https://www.npmjs.com/package/sfml",
+		"Downloads": "3679"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-494",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "N",
+		"AC": "H",
+		"PR": "L",
+		"UI": "R",
+		"S": "U",
+		"C": "H",
+		"I": "H",
+		"A": "H",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "7.1"
+	},
+	"CVEs": [
+		"CVE-2016-10654"
+	],
+	"Repository": {
+		"URL": "https://github.com/bmeck/node-sfml",
+		"Codebase": [
+			"JavaScript"
+		]
+	},
+	"Permalinks": [
```

Comment:
Please add also the other `http` downloads made by the same tool:

* https://github.com/bmeck/node-sfml/blob/master/install#L50
* https://github.com/bmeck/node-sfml/blob/master/install#L69

Cheers,
Mik

Reply:
Again, thank you for the extra references, anything that makes this process more efficient for our users is of great value!

Ben ❤️